vcpkg/concepts/overlay-ports.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Overlay ports
+description: This article describes overlay ports in vcpkg. Overlay ports are used to force a specific port definition to be used by vcpkg during the package installation process.
+author: data-queue
+ms.author: danshaw2
+ms.date: 9/20/2023
+ms.prod: vcpkg
+ms.topic: concept-article
+---
+
+# Overlay ports
+
+An overlay port is a port that can act as a drop-in replacement for an existing port. An overlay port can also be a new port that is otherwise not available in a [registry](../maintainers/registries.md). While [resolving package names](../users/registries.md#package-name-resolution), overlay ports take priority, forcing overlay ports to be selected.
```

Comment:
```suggestion
Usually, vcpkg ports are obtained from [registries](../users/registries.md). It is very likely that most of the ports you install come from the official vcpkg registry at <https://github.com/Microsoft/vcpkg>. vcpkg lets you install ports available to you via the file system, we call these ports, overlay ports.

An overlay port can act as a drop-in replacement for an existing port or as a new port that is otherwise not available in a [registry](../maintainers/registries.md). While [resolving package names](../users/registries.md#package-name-resolution), overlay ports take priority.
```